src/VisualStudio/IntegrationTest/IntegrationTests/Typescript/TypescriptSignatureHelperTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Typescript
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class TypescriptSignatureHelperTest : AbstractTypescriptIntegrationTest
+    {
+        public TypescriptSignatureHelperTest(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(TypescriptCompletionTest))
+        {
+        }
+
+        [Fact]
+        public void ShowSignature()
+        {
+            VisualStudio.Editor.SetText(@"var v = 0;
+v.toExponential(");
+
+            VisualStudio.Editor.InvokeSignatureHelp();
+            VisualStudio.Editor.Verify.CurrentSignature(
```

Comment:
I don't think you need to check the whole signature here, we could break this. Just checking it shows and starts with `toExponential` is probably fine for your purpose and a lot safer.